### ISSUE TITLE
预训练模型对应的训练集或者损失函数权重的设置

### ISSUE THREAD

**Issue by tucker666** [Jeff-sjtu/HybrIK#155]
您好，有个问题想请教下
我这边在重新训练模型时，预加载pose_hrnet_w48_256x192.pth权重，同时加入了新的数据集，导致的结果是在新数据集上表现变好，但是pw3d和h36m数据集的MPJPE分别增加了20到30mm
于是我做了另一个实验，只用原有的数据集训练，配置文件如下，最后得到的指标依旧达不到pose_hrnet_w48_256x192.pth那样好的效果，在零散的测试视频上也没有pose_hrnet_w48_256x192.pth效果好，想请教下是否在训练数据集设置或者损失函数权重上有需要注意的地方，或者需要调整的地方，如能有空解答，非常感谢！
DATASET:
  DATASET: 'mix2_smpl_cam'
  SET_LIST:
    - ROOT: './data/h36m/'
      TEST_SET: 'Sample_20_test_Human36M_smpl'
      TRAIN_SET: 'Sample_trainmin_train_Human36M_smpl_leaf_twist'
    - ROOT: './data/coco/'
      TRAIN_SET: 'train2017'
    - ROOT: './data/3dhp/'
      TRAIN_SET: 'train_v2'
    - ROOT: './data/pw3d/'
      TRAIN_SET: '3DPW_train_new'
  PROTOCOL: 2
  FLIP: True
  ROT_FACTOR: 30
  SCALE_FACTOR: 0.3
  NUM_JOINTS_HALF_BODY: 8
  PROB_HALF_BODY: -1
  COLOR_FACTOR: 0.2
  OCCLUSION: True
MODEL:
  TYPE: 'HRNetSMPLCam'
  HR_PRETRAINED: './pose_hrnet_w48_256x192.pth'
  PRETRAINED: ''
  # TRY_LOAD: ''
  TRY_LOAD: ''
  RESUME: ''
  FOCAL_LENGTH: 1000
  IMAGE_SIZE:
  - 256
  - 256
  HEATMAP_SIZE:
  - 64
  - 64
  HALF: 0
  NUM_JOINTS: 29
  HRNET_TYPE: 48
  EXTRA:
    SIGMA: 2
    BACKBONE: 'resnet'
    CROP: 'padding'
    AUGMENT: 'none'
    PRESET: 'simple_smpl_3d_cam'
    DEPTH_DIM: 64
  POST:
    NORM_TYPE: 'softmax'
  BBOX_3D_SHAPE:
    - 2200
    - 2200
    - 2200
LOSS:
  TYPE: 'LaplaceLossDimSMPLCam'
  ELEMENTS:
    BETA_WEIGHT: 1
    BETA_REG_WEIGHT: 0
    PHI_REG_WEIGHT: 0.0001
    LEAF_REG_WEIGHT: 0
    TWIST_WEIGHT: 1
    THETA_WEIGHT: 0.01
    UVD24_WEIGHT: 1
    XYZ24_WEIGHT: 0
    XYZ_SMPL24_WEIGHT: 0
    XYZ_SMPL17_WEIGHT: 0
    VERTICE_WEIGHT: 0
    USE_LAPLACE: True
TEST:
  HEATMAP2COORD: 'coord'
TRAIN:
  WORLD_SIZE: 4
  BATCH_SIZE: 64
  BEGIN_EPOCH: 0
  END_EPOCH: 20
  OPTIMIZER: 'adam'
  LR: 0.0001
  LR_FACTOR: 0.1
  LR_STEP:
  - 8
  - 12
  DPG_MILESTONE: 14
  DPG_STEP:
  - 16
  - 19
